### DISCUSSION TITLE
Why is fake_bg totally wrong using StyleText when the input image is 320*32?

### DISCUSSION THREAD

**Discussion by Greenhand818** [PaddlePaddle/PaddleOCR#6609] [Q&A] [answered]
This is my input.![resized_4](https://user-images.githubusercontent.com/75562954/174024191-322d0f61-a2cb-408b-a510-a78a15efccfd.jpg)

This is fake_bg.![fake_bg](https://user-images.githubusercontent.com/75562954/174024541-4c42ae6f-f77d-4afa-9bf8-1383d15aa8fa.jpg)

This is fake_fusion.![fake_fusion](https://user-images.githubusercontent.com/75562954/174024655-f36429c7-2a42-4bf7-bdab-fef9820909e5.jpg)

**Comment by weisy11** [comment, marked as answer]
When the text size is too large and background info is not enough, it will be difficult for the model to extract bg_image. Please crop more background and try again.
